### ISSUE TITLE
Using functions with comma throws an error

### ISSUE THREAD

**Issue by yannychn** [forceedge01/genesis-behat-sql-extension#14]
An example:

date:DATE_SUB(NOW(), INTERVAL 2 DAY)

Error:

Unable to explode columns based on ":" separator

**Comment by forceedge01** [comment]
Thanks @yannychn , will try to get onto this. For now I think using a function without comma's is the only way.

**Comment by petetrickey** [comment]
@forceedge01 This is also an issue for adding any content to a field with either comma's or colon's.

Issue appears to be in SQLHandler::filterAndConvertToArray()

I should have a "database.table" with "name: Joe Bloggs, address_1: My House, street_address: 1, my road, town_or_city: My city, district: My district, postcode: AB12 3DE"

It's the following section that causes the failure.

street_address: 1, my road

**Comment by jpswade** [comment]
http://php.net/manual/en/function.str-getcsv.php

**Comment by forceedge01** [comment]
Thanks for more information on this bug, this has been fixed in version 2.0.3 - you should be able to escape the comma character "," with a backslash. 
@yannychn for your example this will become:
```
date:DATE_SUB(NOW()\, INTERVAL 2 DAY)
```

@petetrickey for your scenario the solution right now would be:
```
I should have a "database.table" with "name: Joe Bloggs, address_1: My House, street_address: 1\, my road, town_or_city: My city\, district: My district, postcode: AB12 3DE"
```

Please note the use of backslashes in different places. Let me know if something else comes up, for now this issue is being closed off.
